### DISCUSSION TITLE
Alert mailing not working while sql query mailing works

### DISCUSSION THREAD

**Discussion by simsicon** [apache/dolphinscheduler#8003] [Q&A]
Hi, I have dolphinscheduler 1.3.9 deployment on a k8s cluster, and I think I have configured the smtp server correctly because I can receive sql query emails, but I can not receive alert mails no matter how I changed the sending strategies. I am aware of the notification strategy(通知策略)/notification group(通知组)/receiver(收件人) settings involves the alert mailing.

And I can not find any useful information in dolphinscheduler-alert log, what could possibly be wrong with my configuration? How can I debug it? Thanks

**Comment by simsicon** [comment]
@dailidong would you mind take a look?

**Reply by SharkAndCat** [reply]
How do you configure it? I also encountered a similar problem, but I could only receive alarm emails and could not receive SQL query results

**Reply by simsicon** [reply]
I use dolphinscheduler 3 now.

**Reply by SharkAndCat** [reply]
You've had no problems with dolphinscheduler 3? I use 3.1.7, there is such a problem, how do you configure it? Files that can receive query results
